modules/nw-rfhe-creating_bmc_event_sub.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
```

Comment:
This says networking/using-rfhe.adoc but I see it under monitoring/using-rfhe.adoc

Reply:
updated